SmartDeviceLinkTests/RPCSpecs/StructSpecs/SDLKeyboardPropertiesSpec.m

```diff
@@ -32,14 +32,12 @@
         testStruct.keyboardLayout = testLayout;
         testStruct.keypressMode = testMode;
         testStruct.limitedCharacterList = testLimitedCharacterList;
-        testStruct.autoCompleteText = testAutoCompleteText;
```

Comment:
All the tests for `autoCompleteText` should be added back. We are marking `autoCompleteText` as deprecated but developers should still be able to use it (i.e. developers will still need to use `autoCompleteText` when connecting to older modules).

Reply:
All the tests for `autoCompleteText` still need to be added back